notebooks/random_forest_clean.ipynb

```diff
@@ -0,0 +1,1507 @@
+{
```

Comment:
<p>this random shuffling is not controlled by a hard-coded seed; thus not reproducible.  Every time the notebook is re-run, there is a different train and test split, and thus different validation results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>added random.seed() command</p>